README.md

```diff
@@ -48,8 +54,8 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         signoff: true # optional
-        author: update-bot # optional
-        committer: update-bot@example.com # optional
+        author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com> # optional
+        committer: GitHub <noreply@github.com> # optional
```

Comment:
Optional bit of magic we do in setup-gitsign: https://github.com/chainguard-dev/actions/blob/03695c5514eaccea19c5b138261568318d5b3be7/setup-gitsign/action.yml#L110-L111

If you use `41898282+github-actions[bot]@users.noreply.github.com` as the author/committer (the important bit is `41898282`, you can replace `github-actions[bot]` with whatever you want), the commit will render as GitHub Actions - https://github.com/wlynch/test/commit/a1da74af3f3491f520d625fa35159f20bf9ec99c

Reply:
i always get confused with that, but you always come to save me :)

will update